ext/CEGUI/CEGUI.cpp

```diff
@@ -97,6 +98,15 @@ void GUIManager::init()
     ));
 }
 
+void GUIManager::render()
+{
+    auto renderState = irr::video::COpenGLState::collectGLState();
```

Comment:
`collectGLState` and `executeGLDiff` take a list of booleans about what state it should care about.

CEGUI uses OpenGL 3 so things like SSBOs, Storage Images, etc. shouldn't matter

Also if you care about current FBO, Viewport, etc then it will change those, sometimes that is not what you want, this is why in the silverlining example I told it to not care about FBO, depth test, etc. etc. because I wanted to render it into a texture with depth testing.

Reply:
Added to task list, will review this when things render.

Reply:
done in latest commit